frontend/src/api/k8s/servingRuntimes.ts

```diff
@@ -79,11 +79,24 @@ const assembleServingRuntime = (
 
   const { affinity, tolerations, resources } = assemblePodSpecOptions(resourceSettings, gpus);
 
+  const volumes = updatedServingRuntime.spec.volumes || [];
+  // Can we assume there is only 1 container per serving runtime for now?
+  const volumeMounts = updatedServingRuntime.spec.containers?.[0].volumeMounts || [];
```

Comment:
I wouldn't assume that, with Custom Serving Runtimes we can have multiple possibilities, you are even adding this configuration for several containers right bellow.

Reply:
If we don't make the assumption, how can we locate which container we are going to fetch and add the `VolumeMounts`?

Reply:
mmmm there are a couple of things to consider here, I'm gonna ping @erwangranger in case there's something wrong:

1. `shm` is just a naming convention, you can have a volume called `dshm` or whatever other name
2. The important part is that whatever volume you have defined with whatever name, it's mounted then in a container in `/dev/shm`

So I would do the following, considering that this is for default Serving Runtimes that doesn't consider this:

1. Check if a volume called `shm` exists, if not, create that with the value we decided (`2Gi`)
2. Then iterate through all the containers, if they contain a `volumeMount` with `mountPath` pointing at `/dev/shm` leave it as it is, if not, add a `volumeMount` to the `volumeMount` array of the container.

Reply:
I really am not sure. I think @guimou or @cfchase  probably understand this better than I do. (can you please review and comment?)
Even the `2 Gi` was just a value I chose arbitrarily, but I really don't have any idea if that is reasonable, or if it should be something the admin might want to update.